packages/expo-notifications/ios/EXNotifications/Notifications/Scheduling/EXNotificationSchedulerModule.m

```diff
@@ -110,6 +115,16 @@ - (UNNotificationTrigger *)triggerFromParams:(NSDictionary *)params
 
     return [UNTimeIntervalNotificationTrigger triggerWithTimeInterval:[date timeIntervalSinceNow]
                                                               repeats:NO];
+
+  } else if ([dailyIntervalNotificationTriggerType isEqualToString:triggerType]) {
+    NSNumber *hour = [params objectForKey:dailyIntervalNotificationTriggerHourKey verifyingClass:[NSNumber class]];
+    NSNumber *minute = [params objectForKey:dailyIntervalNotificationTriggerMinuteKey verifyingClass:[NSNumber class]];
+    NSDateComponents *dateComponents = [NSDateComponents new];
+    dateComponents.hour = [hour integerValue];
+    dateComponents.minute = [minute integerValue];
```

Comment:
Yes, addressed above. I think it will be best to handle this on the JS side as the valid ranges of hour and minute do not depend on the platform. 

Not validating the input makes this PR sure seem like 

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia2.giphy.com%2Fmedia%2F3DzwbT4oF52JG%2Fgiphy.gif&token=amateur" data-gitmeme-token="amateur"><img src="https://media2.giphy.com/media/3DzwbT4oF52JG/giphy.gif" title="Created by gitme.me with /amateur"/></a>